build.gradle

```diff
@@ -4,18 +4,20 @@ buildscript {
     repositories {
         jcenter()
         mavenCentral()
+        google()
     }
     dependencies {
-	    classpath 'com.android.tools.build:gradle:2.2.3'
+	    classpath 'com.android.tools.build:gradle:3.0.0-beta5'
```

Comment:
This is a move in the wrong direction.
Even we are forced to use targetSdk > 26 because of https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html till November 2018.

Reply:
The guys who want to stay at 2.2.3 should simply not update the lib